front/src/components/ScheduleAndShiftsCreationComponents/SaveCurrentlyEditingShift.ts

```diff
@@ -0,0 +1,22 @@
+import {ShiftMetadata} from "./Types.ts";
+import {Guid} from "guid-typescript";
+
+export type ShiftDataToSave = {
+    editingShiftStartTime: Date;
+    editingShiftEndTime: Date;
+    shiftInScheduleToUpdate: ShiftMetadata;
+    callBack: (shiftId: Guid, startTime: Date, endTime: Date) => void;
+}
+
+export const saveCurrentlyEditingShift = (editingShiftDataToSave: ShiftDataToSave) => {
```

Comment:
remove func.
extract time resetting function with good name
give better name to start time